spec/controllers/courses_controller_spec.rb

```diff
@@ -140,6 +140,25 @@
       end
     end
 
+    it 'updates the wikis' do
+      course_params[:wikis] = [{ language: 'de', project: 'wikipedia' }]
```

Comment:
The idea here is that this is testing the server-side handling of an update where the wikis don't include the home_wiki? So even though only one wiki was received, we expect the home wiki to also be there? If so, that's not obvious enough from the test itself.

I recommend either adding comments, or separating these into four different tests with titles that explain exactly what each one is trying to test for.